sc-tools/sc-config/tests/units/test_config.cpp

```diff
@@ -83,5 +85,74 @@ TEST(ScConfig, DeprecatedExtensionsPath)
   EXPECT_EQ(std::string(params.log_file), "");
   EXPECT_EQ(std::string(params.log_level), "Debug");
   EXPECT_EQ(std::string(params.storage), "sc-machine-test-repo");
-  EXPECT_EQ(std::string(params.extensions), "bin/extensions");
+  EXPECT_EQ(params.extensions_directories_count, 1u);
+  EXPECT_EQ(std::string(params.extensions_directories[0]), "bin/extensions");
+}
+
+TEST(ScConfig, Multiextensions)
+{
+  ScConfig config{SC_CONFIGS "/multiextensions.ini"};
+
+  ScMemoryConfig memoryConfig{config, {}};
+
+  sc_memory_params const params = memoryConfig.GetParams();
+  EXPECT_EQ(params.max_loaded_segments, 1000u);
+  EXPECT_EQ(params.dump_memory, SC_TRUE);
+  EXPECT_EQ(params.dump_memory_period, 4u);
+  EXPECT_EQ(params.dump_memory_statistics, SC_TRUE);
+  EXPECT_EQ(params.dump_memory_statistics_period, 4u);
+  EXPECT_EQ(std::string(params.log_type), "Console");
+  EXPECT_EQ(std::string(params.log_file), "");
+  EXPECT_EQ(std::string(params.log_level), "Debug");
+  EXPECT_EQ(std::string(params.storage), "sc-machine-test-repo");
+  EXPECT_EQ(params.extensions_directories_count, 3u);
+  EXPECT_EQ(std::string(params.extensions_directories[0]), "bin/extensions_1");
+  EXPECT_EQ(std::string(params.extensions_directories[1]), "bin/extensions_2");
+  EXPECT_EQ(std::string(params.extensions_directories[2]), "bin/extensions_3");
+}
+
+TEST(ScConfig, EmptyMultiextensions)
+{
+  ScConfig config{SC_CONFIGS "/empty-multiextensions.ini"};
+
+  ScMemoryConfig memoryConfig{config, {}};
+
+  sc_memory_params const params = memoryConfig.GetParams();
+  EXPECT_EQ(params.max_loaded_segments, 1000u);
+  EXPECT_EQ(params.dump_memory, SC_TRUE);
+  EXPECT_EQ(params.dump_memory_period, 4u);
+  EXPECT_EQ(params.dump_memory_statistics, SC_TRUE);
+  EXPECT_EQ(params.dump_memory_statistics_period, 4u);
+  EXPECT_EQ(std::string(params.log_type), "Console");
+  EXPECT_EQ(std::string(params.log_file), "");
+  EXPECT_EQ(std::string(params.log_level), "Debug");
+  EXPECT_EQ(std::string(params.storage), "sc-machine-test-repo");
+  EXPECT_EQ(params.extensions_directories_count, 0u);
+  EXPECT_EQ(params.extensions_directories, nullptr);
+}
+
+TEST(ScConfig, NotNormalizedMultiextensions)
+{
+  ScConfig config{SC_CONFIGS "/../tests/not-normalized-multiextensions.ini", {"extensions"}};
+
+  ScMemoryConfig memoryConfig{config, {}};
+
+  sc_memory_params const params = memoryConfig.GetParams();
+  EXPECT_EQ(params.max_loaded_segments, 1000u);
+  EXPECT_EQ(params.dump_memory, SC_TRUE);
+  EXPECT_EQ(params.dump_memory_period, 4u);
+  EXPECT_EQ(params.dump_memory_statistics, SC_TRUE);
+  EXPECT_EQ(params.dump_memory_statistics_period, 4u);
+  EXPECT_EQ(std::string(params.log_type), "Console");
+  EXPECT_EQ(std::string(params.log_file), "");
+  EXPECT_EQ(std::string(params.log_level), "Debug");
+  EXPECT_EQ(std::string(params.storage), "sc-machine-test-repo");
+  EXPECT_EQ(params.extensions_directories_count, 3u);
+
+  std::string const & prefix = "../../../../sc-tools/sc-config/tests/";
```

Comment:
I wanted to check why this happens but I couldn't build with -t and I couldn't make_tests.sh because this happens, how can I fix it?
![image](https://github.com/user-attachments/assets/1930fca2-d0ea-4cee-9b9c-c3937074341a)
this path should be relative to current_dir(just '..'), but it makes 4 parent directories and then goes back to tests dir

Reply:
git submodule update --init --recursive